.github/workflows/test.yml

```diff
@@ -36,6 +36,7 @@ env:
   APIM_MDM_KEY: ${{ secrets.APIM_MDM_KEY }}
   APIM_MDM_VALUE: ${{ secrets.APIM_MDM_VALUE }}
   GOOGLE_ANALYTICS_ID: ${{ secrets.GOOGLE_ANALYTICS_ID }}
+  GOOGLE_TAG_MANAGER_ID: ${{ secrets.GOOGLE_TAG_MANAGER_ID }}
```

Comment:
do we need it in tests?

Reply:
Yes, it's used in tests for the cookies constent middleware